packages/documentation-v7/src/stories/internet-header/components/internet-header.docs.mdx

```diff
@@ -0,0 +1,54 @@
+import { Canvas, ArgTypes, Meta } from '@storybook/blocks';
+import * as InternetHeaderStories from './internet-header.stories';
+
+<Meta of={InternetHeaderStories} />
+
+# Header
+
+<div className="lead">
+  Customizable navigation header tailored for Swiss Post customer-facing applications,
+  ensuring complete accessibility and responsiveness.
+</div>
+
+The `<swisspost-internet-header>` element needs
+to be nested directly inside the `<body>` for the sticky scrolling to work.
+
+If you want to render the header element with Angular,
+have a look at [portals](https://medium.com/angular-in-depth/how-do-cdk-portals-work-7c097c14a494).
+
+<a
+  className="btn btn-primary btn-animated"
+  href="/iframe.html?viewMode=story&id=internet-header-header-component--default"
+  target="_blank"
+>
+  <span>Open Header in New Tab</span>
+</a>
+
+<Canvas of={InternetHeaderStories.Default} />
+
+<div className="hide-col-default">
+  <ArgTypes of={InternetHeaderStories.Default} />
```

Comment:
I used `ArgTypes` instead of `Controls` here because [controls don't work on iframe demos](https://github.com/storybookjs/storybook/issues/11908), but an iframe is needed for the logo animation to work properly.